src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+
+  if (days >= 7) {
+    return days * dayPrice - 50;
+  }
```

Comment:
Prefer using 'const' for the 'dayPrice' if you don't plan to reassign it to ensure the value doesn't get changed accidentally later in the code.